src/sentry/deletions/base.py

```diff
@@ -238,6 +239,14 @@ class BulkModelDeletionTask(ModelDeletionTask):
     """
     DEFAULT_CHUNK_SIZE = 10000
 
+    def __init__(self, manager, model, query, chunk_size=DEFAULT_CHUNK_SIZE,
+                 partition_key=None, **kwargs):
```

Comment:
This also fixes it so that `DEFAULT_CHUNK_SIZE = 10000` actually works.

Reply:
Let's just move to it's own PR just so we have less moving bits. Going from 100 to 10000 is a big difference.